codacy/templates/fluentd/fluentd-conf.yaml

```diff
@@ -17,48 +17,63 @@ data:
     <filter **>
       @type record_modifier
       <record>
-        # This removes unneeded fields from the logs
         for_remove ${record["kubernetes"].delete("namespace_name"); record["kubernetes"].delete("pod_id"); record["kubernetes"].delete("namespace_labels"); record["kubernetes"].delete("container_info"); record["kubernetes"].delete("labels"); record.delete("container_info"); record.delete("docker")}
       </record>
+      # This removes unneeded fields from the logs
```

Comment:
What fields are unneeded and why?

Reply:
No idea. I just moved the comment to the right line where they are removed. 
The line where the comment was before is just assigning a new name to those keys from my understanding

Reply:
```suggestion
      # This removes the fields defined in `for_remove` from the logs
```

Reply:
Thanks for the inline fix